Introductory Tests/udp_class.py

```diff
@@ -0,0 +1,93 @@
+# General
+import numpy as np
+
+# For computing acceleration and potential
+import polyhedral_gravity as model
+
+# For working with the mesh
+import mesh_utility
+
+# For Plotting
+import pyvista as pv
+
+class udp_obj:
+    
+    def __init__(self,density, r_T, t_end, t_0, dt, lb, ub):
+        # Creating the mesh (TetGen)
+        self.mesh, self.vertices, self.faces = mesh_utility.create_mesh()
+
+        # Additional hyperparameters
+        self.density = density     
+        self.r_T = r_T     
+        self.t_end = t_end      
+        self.t_0 = t_0                
+        self.dt = dt
+        self.lb = lb
```

Comment:
lower_bounds (avoid abbreviations ideally, helps comprehensibility)
same below for upper_bounds